buf/registry/module/v1/label_service.proto

```diff
@@ -175,6 +178,9 @@ message ListLabelHistoryResponse {
     Commit commit = 1 [(buf.validate.field).required = true];
     // The CommitCheckState for this Commit on this Label.
     CommitCheckState commit_check_state = 2 [(buf.validate.field).required = true];
+    // Whether or not the content of this Commit is different
+    // compared to the previous Commit in the history of this Label.
+    bool content_change = 3;
```

Comment:
Shouldn't you be able to just compare the digest of the previous commit in your response? This is duplicate info otherwise.

Reply:
whoops, you are correct we don't need this. Removed.